pkg/reconciler/reconciler.go

```diff
@@ -673,6 +700,10 @@ func (r *Reconciler) doUpgrade(actionClient helmclient.ActionInterface, u *updat
 		}
 	}
 
+	log.Info("Upgrade config",
+		"name", obj.GetName(), "namespace", obj.GetNamespace(), "values", vals, "options", opts)
+
+	// todo: the upgrade release no longer is returning the previous value for we diff
 	rel, err := actionClient.Upgrade(obj.GetName(), obj.GetNamespace(), r.chrt, vals, opts...)
```

Comment:
@joelanford the actionClient.Upgrade is not longer return the previous release for we diff. see: https://github.com/operator-framework/operator-sdk/blob/c703c8d04e94ff6f930ad04a52cc5262141df79f/pkg/helm/controller/reconcile.go#L266-L268